listenbrainz/influx-writer/influx-writer.py

```diff
@@ -191,18 +196,32 @@ def write(self, listen_dicts):
             timestamps = {}
             for result in results.get_points(measurement=get_measurement_name(user_name)):
                 dt = datetime.strptime(result['time'] , "%Y-%m-%dT%H:%M:%SZ")
-                timestamps[int(dt.strftime('%s'))] = 1
+                timestamps[int(dt.strftime('%s'))] = result
 
             for listen in users[user_name]['listens']:
-                # Check to see if the timestamp is already in the DB
                 t = int(listen['listened_at'])
-                if t in timestamps:
-                    duplicate_count += 1
-                    continue
-
-                unique_count += 1
-                submit.append(Listen().from_json(listen))
-                unique.append(listen)
+                artist_msid    = listen['track_metadata']['additional_info']['artist_msid']
```

Comment:
Ahh, this is an msid, not an mbid. So, it'll be set (because artist names are required)